apps/dashboard/app/views/layouts/nav/_link.html.erb

```diff
@@ -13,18 +13,21 @@ data - optional, data parameter for the link_to helper
   url = local_assigns.fetch(:url)
   icon_uri = local_assigns.fetch(:icon_uri, URI("fas://cog"))
 %>
+<li role="none">
 <%=
   link_to(
     url,
     title: title,
-    class: "#{local_assigns.fetch(:class, 'nav-link')} #{local_assigns.fetch(:role, nil)}",
+    class: "#{local_assigns.fetch(:class, 'nav-link')}",
```

Comment:
The role was being added to the class for some reason? In any case, all these have the same role, so this is being removed from the `nav_link` api anyhow.